### PR TITLE
Ecto prototype

### PR DESCRIPTION
**Pull request by LtChae**
This is a draft and may not be committed to master as-is. Pending coordination with the pair working on smartcitiesdata/discovery_ui#203

### CODE REVIEW

File: lib/discovery_api/application.ex [hunk 1]
```diff
@@ -20,12 +20,14 @@ defmodule DiscoveryApi.Application do
...
         supervisor(DiscoveryApiWeb.Endpoint, []),
-        DiscoveryApi.Quantum.Scheduler
+        DiscoveryApi.Quantum.Scheduler,
+        postgres()
       ]
       |> List.flatten()
 
     opts = [strategy: :one_for_one, name: DiscoveryApi.Supervisor]
     Supervisor.start_link(children, opts)
+    # migrate()
```
Comment (jdenen) [inline lib/discovery_api/application.ex]: Should this be deleted? Why are we keeping it?

File: lib/discovery_api/data/postgres.ex [hunk 2]
```diff
@@ -0,0 +1,46 @@
+defmodule DiscoveryApi.Repo do
```
Comment (jdenen) [inline lib/discovery_api/data/postgres.ex]: This doesn't feel like the right subdirectory/namespace for these modules.

Data is all about datasets and their contents, not generally data that `discovery_api` deals with. Should we create a `Users` context?

File: lib/discovery_api/application.ex (hunk 1 above)
Comment (LtChae) [reply lib/discovery_api/application.ex]: This is a prototype that we made for Tony's card. This isn't getting into master, you'll see a version of it in his changes.

File: start.sh [hunk 3]
```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+bin/discovery_api migrate
```
[the remark this answers is not in the document]
Comment (LtChae) [reply start.sh]: Seems to be, yeah. I found a couple of articles that suggested variants of this

**Pull request closed without merging**